dhcp-relay/dhcp_kern_xdp.c

```diff
@@ -499,7 +685,6 @@ int xdp_dhcp_relay(struct xdp_md *ctx) {
 	/* Re-calculate ip checksum */
 	__u32 sum = calc_ip_csum(&oldip, ip, oldip.check);
 	ip->check = ~sum;
-	rc = XDP_PASS;
```

Comment:
See above; this *is* actually the right place to change the return code back into something that sends out the packet, as this is where the packet is once again valid...